src/stages/appflow-ingestion.ts

```diff
@@ -112,11 +111,8 @@ export class AppFlowIngestionStage extends StateMachineStage {
       description: "lambda role to check appflow flow execution status",
     });
 
-    const statusLambda = new lambda.Function(
-      this,
-      "Flow Execution Status Lambda",
-      LambdaDefaults.functionProps({
-        code: lambda.Code.fromInline(`
+    const statusLambda = new lambda.Function(this, "Flow Execution Status Lambda", {
```

Comment:
Just curious, what's the reason for not having a Lambda factory?

Reply:
I left it out as it was only setting `memorySize: 128 -> 256` & `timeout: 3 -> 120`, which doesn't accelerate experience much for the user. I'm open to adding it back though. I think the rest I had left out had no defaults.